hk2-locator/src/test/java/org/glassfish/hk2/tests/locator/negative/factory/BadlyNamedFactory.java

```diff
@@ -20,21 +20,22 @@
 import javax.inject.Singleton;
 
 import org.glassfish.hk2.api.Factory;
+import org.glassfish.hk2.api.PerLookup;
 
 /**
  * This factory has a provide method with a {@link Named}
  * annotation with no value, which is illegal
- * 
+ *
  * @author jwells
  *
  */
-@Singleton
+@PerLookup
```

Comment:
Why is this changed?

Reply:
It is not allowed annotation here. But as exceptions were swallowed, it was ignored. This annotation was not tested, the missing name atttribute was.